sql-plugin/src/main/scala/com/nvidia/spark/rapids/basicPhysicalOperators.scala

```diff
@@ -408,13 +409,16 @@ case class GpuUnionExec(children: Seq[SparkPlan]) extends SparkPlan with GpuExec
   override def doExecuteColumnar(): RDD[ColumnarBatch] = {
     val numOutputRows = gpuLongMetric(NUM_OUTPUT_ROWS)
     val numOutputBatches = gpuLongMetric(NUM_OUTPUT_BATCHES)
+    val opTime = gpuLongMetric(OP_TIME)
     val totalTime = gpuLongMetric(TOTAL_TIME)
 
-    sparkContext.union(children.map(_.executeColumnar())).map { batch =>
-      withResource(new NvtxWithMetrics("Union", NvtxColor.CYAN, totalTime)) { _ =>
-        numOutputBatches += 1
-        numOutputRows += batch.numRows
-        batch
+    withResource(new NvtxWithMetrics("Union", NvtxColor.PURPLE, totalTime)) { _ =>
+      sparkContext.union(children.map(_.executeColumnar())).map { batch =>
+        withResource(new NvtxWithMetrics("Union", NvtxColor.CYAN, opTime)) { _ =>
```

Comment:
Does it make sense to have an operation timing for this?  This is literally performing two sums on the CPU, so any value reported will be dominated by metric measurement overhead and not be useful in practice.

Reply:
It does seem pretty pointless. I will remove it.